docs/docs/installation.md

```diff
@@ -218,18 +218,18 @@ Learn [how to tell if your website is serving the right version of React](/react
 If you don't want to use npm to manage client packages, the `react` and `react-dom` npm packages also provide single-file distributions in `dist` folders, which are hosted on a CDN:
 
 ```html
-<script crossorigin src="https://unpkg.com/react@15/dist/react.js"></script>
-<script crossorigin src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+<script crossorigin src="https://unpkg.com/react@16/dist/react.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/dist/react-dom.js"></script>
 ```
 
 The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
 
 ```html
-<script crossorigin src="https://unpkg.com/react@15/dist/react.min.js"></script>
-<script crossorigin src="https://unpkg.com/react-dom@15/dist/react-dom.min.js"></script>
+<script crossorigin src="https://unpkg.com/react@16/dist/react.min.js"></script>
```

Comment:
Broken URLs. We probably need to update them everywhere.

Reply:
(Because there is no /dist anymore. UMD builds are in /umd)

Reply:
Oops